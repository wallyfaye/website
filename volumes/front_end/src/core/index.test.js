import React from 'react';
import Core from './index';
import renderer from 'react-test-renderer';

test('Link changes the class when hovered', () => {
  const component = renderer.create(
    <Core />,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
