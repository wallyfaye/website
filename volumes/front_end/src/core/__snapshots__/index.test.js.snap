// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Core should render 1`] = `
<div>
  Website
</div>
`;
